.github/workflows/check-coverage.yml

```diff
@@ -5,8 +5,12 @@ name: Unit tests & Coverage
 on:
   push:
     branches: [ main ]
+    paths-ignore:
+      - 'doc/**'
```

Comment:
Maybe we don't want to ignore the docs directory here. This workflow runs the netplan tests and one of the things it does is to check if the configuration keywords present in the code are also present in the docs. So we'd like to know if doc changes removed one of these keywords.

Reply:
Ack. I removed the ignores.